app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -125,24 +133,24 @@ export class SemiCircle extends React.Component {
     100% { offset-distance : ${arcEndPercent}% }
     `;
 
+    const styleSemiCircle = () => {
+      if (nightMode) return colors.lightBlue;
+      if (cityImage === bldImage) return colors.lightYellow;
```

Comment:
This should be its own refactor commit to extract the function and then add the boulder condition as part of the boulder commit.

Reply:
This should be fixed now!